internal/handshake/session_ticket.go

```diff
@@ -37,11 +41,13 @@ func (t *sessionTicket) Unmarshal(b []byte) error {
 	if err != nil {
 		return errors.New("failed to read RTT")
 	}
-	var tp wire.TransportParameters
-	if err := tp.UnmarshalFromSessionTicket(r); err != nil {
-		return fmt.Errorf("unmarshaling transport parameters from session ticket failed: %s", err.Error())
+	if r.Len() > 0 {
```

Comment:
What do you think of passing a 0-RTT bool to this function. This would make the parsing logic stricter.

Reply:
That sounds reasonable because the two kinds of ticket should be parsed with different logic and unmarshaling a 0-RTT ticket without transport parameters should fail. I have added it.